infrasec/aws/vpcs.md

```diff
@@ -0,0 +1,82 @@
+# [AWS](./README.md) / VPC Configuration
+
+## VPC basics
+
+A VPC is a "Virtual Private Cloud".
+This is a virtual network defined in AWS.
+Each VPC is defined in a specified [region](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions).
+
+The VPC needs a defined CIDR block.
+These are private network blocks and need to have enough ip addresses for whatever you plan to deploy.
+When in doubt, use a `/16`.
+
+Note that each region has multiple [Availability Zones](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-availability-zones) and the ip space will be subdivided into subnets across these availabilty zones.
+
+The following are objects associated with VPCs that can be configured.
+
+### Subnets
+
+A subnet is a logical subdivision of an IP network.
+Generally, you will create a pair of public and private subnets in each availability zone for the region.
+A good default CIDR range for each subnet is either a `/22` or `/24`.
+
+Creating additional subnets will create additional complexity in route tables.
+If your project requires you to isolate resources from one another, use Security groups.
```

Comment:
At OneMedical, they initially reused subnets in various places and Jeremy suggested they would regret it.  By the time I was there, they were regretting it.  Perhaps call out reuse can be a problem?

Reply:
Good point. Going to add that.